src/parsers/init.js

```diff
@@ -0,0 +1,24 @@
+class BaseParser {
```

Comment:
There should StructurParser with `cell`, `elements` etc. as an intermediate data